content/2_Bronze/Complete_Rec.mdx

```diff
@@ -561,32 +524,29 @@ smaller than the initial string won't be generated.
 </Warning>
 
 ```cpp
-#include <iostream>
-#include <vector>
-#include <algorithm>
+#include <bits/stdc++.h>
+
 using namespace std;
 
-int main()
-{
+int main() {
 	string s;
 	cin >> s;
 	vector<string> perms;
 	sort(s.begin(), s.end());
-	do
-	{
+	do {
 		perms.push_back(s);
-	}
-	while (next_permutation(s.begin(), s.end()));
+	} while (next_permutation(s.begin(), s.end()));
 	cout << perms.size() << endl;
-	for (string perm : perms)
-	{
+	for (string perm : perms) {
 		cout << perm << endl;
 	}
 }
 ```
 
 </CPPSection>
 
+<JavaSection/>
```

Comment:
what's this for

Reply:
i believe it hides the uncompleted message for java - java has no built-in method to generate permutations